plugins/services/services_inspector.rb

```diff
@@ -106,8 +118,43 @@ def inspect_upstart_services
     services += disabled.map(&:first).each.map do |name|
       Service.new(name: name, state: "disabled")
     end
+  end
 
-    services.sort_by(&:name)
+  def parse_ubuntu_sysv
+    # Get all sysV services
+    out, err = @system.run_command(
+      "/usr/sbin/service",
+      "--status-all",
+      stdout: :capture, stderr: :capture)
+    services_output = out + err
+
+    sysv_all = services_output.each_line.map { |line|
+      line.chomp.sub(/^.*\]../, "")
+    }
+
+    # Get all enabled sysV services
+    runlevels = ["2", "3", "4", "5", "S"]
+    find_output = runlevels.each.map do |runlevel|
+      @system.run_command(
+        "/usr/bin/find",
+        "/etc/rc#{runlevel}.d",
+        "-name",
+        "S\*",
+        stdout: :capture).split
+    end
+
+    sysv_enabled = find_output.flatten.map { |line|
```

Comment:
I thought this was our approach for sysv services in other distributions as well, if it gets started in one of the start runlevels, then we consider it as enabled.